lib/greet.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = exports = {};
+var name = process.argv[2];
+
+exports.greet = function(name) {
+  if (arguments.length === 0) throw new Error ('name not included');
+  console.log(`hello ${name}`);
+  return (`hello ${name}`);
+};
+exports.favPlace = function(place) {
+  if (arguments.length === 0) throw new Error ('place not included');
+  return (`your favorite place is ${place}`);
+};
+
+exports.greet(name);
```

Comment:
Be sure to pass favPlace( ) an argument when you invoke in index.js since it's expecting to be passed a place.